### PR TITLE
Checking if the fancy codecov works with java too

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

Related to #187

### CODE REVIEW

File: .github/workflows/knative-test.yaml [hunk 1]
```diff
@@ -57,9 +62,14 @@ jobs:
...
         run: go test -race $COVER_OPTS ./...
 
+      - name: Java Test
+        run: mvn verify --file data-plane/pom.xml --no-transfer-progress
+
       - if: steps.codecov-enabled.outputs.files_exists == 'true'
         name: Codecov
         uses: codecov/codecov-action@v1
+        with:
+          files: ./coverage.xml,./data-plane/target/jacoco/jacoco.xml
```
Comment (pierDipi) [inline .github/workflows/knative-test.yaml:75]: Should we add `coverage.txt` for Go?

Comment (slinkydeveloper) [reply .github/workflows/knative-test.yaml:75]: oh yeah because golang emits txt -_-

**Comment by pierDipi** [comment]
Should we use a separate action for Java since this action is synced by Matt's bot?

**Comment by slinkydeveloper** [comment]
> Should we use a separate action for Java since this action is synced by Matt's bot?

But then we upload 2 reports for each action, does codecov like it?

**Comment by slinkydeveloper** [comment]
woo woo https://codecov.io/gh/knative-sandbox/eventing-kafka-broker/tree/8ec21d0d628e95dff01cc2dd0e14751384307c9b

**Review by pierDipi — APPROVED** [review]
/lgtm

**Pull request merged**